Adyen/Model/Checkout/BrowserInfo.cs

```diff
@@ -283,21 +283,21 @@ public override int GetHashCode()
         IEnumerable<System.ComponentModel.DataAnnotations.ValidationResult> IValidatableObject.Validate(ValidationContext validationContext)
         {
```

Comment:
I think it is better to remove the validation body or the validation function.

Reply:
yes looks good :)

Reply:
Browser info comes from the checkout component so there is no point for the model to make an extra validation